### PR TITLE
(update) code owners

### PR DESCRIPTION
**Pull request by sjaghori**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
-*       @sjaghori
-*       @benjaminstrasser
-*       @jjnp
-*       @mledl
\ No newline at end of file
+*                           @benjaminstrasser @jjnp @mledl @sjaghori
+docs/*                      @sjaghori
+e2e/*                       @benjaminstrasser @sjaghori
+src/components/ui/*         @sjaghori
+src/components/ui/sidebar   @benjaminstrasser
```
Comment (mledl) [inline .github/CODEOWNERS:9]: What is the benefit for this project when splitting the code ownership like that? I do not see it.

Comment (sjaghori) [reply .github/CODEOWNERS:9]: To split the reviews, I'm open for suggestions.

Comment (mledl) [reply .github/CODEOWNERS:9]: Yeah go with no until we have workspaces, packages or adapters.

Comment (sjaghori) [reply .github/CODEOWNERS:9]: let's do everyone for everything

**Review by mledl — APPROVED** [review]

**Pull request merged**